content/news/playtest-20150102.md

```diff
@@ -0,0 +1,34 @@
+We wish everyone a happy new year and prepared a new playtest which builds upon our [Halloween release](/news/release-20141029/). The main focus was to improve stability and performance as well as implement unique features of Dune 2000 that distinguish it from the other mods.
+
+
+These playtest builds provide a snapshot of our current development for feedback by our testers. You can help shape the future of OpenRA by testing these builds and [reporting bugs](http://bugs.openra.net) or simply letting the [community](/community/) know what you think.
+
+Some highlights in this development build are:
+
```

Comment:
"The server browser will now list games from all compatible mods." is another big point that should be included.